assets/static/js/dataset-submission.js

```diff
@@ -258,16 +258,14 @@ $(function() {
             },
             error: function(jqXHR, textStatus, errorThrown) {
                 var message = jqXHR.responseJSON == null ? errorThrown: jqXHR.responseJSON.message;
-                if (notify) {
-                    var n = noty(
-                    {
-                        layout: "top",
-                        theme: "relax",
-                        type: "error",
-                        text: message,
-                        modal: true,
-                    });
-                }
+                var n = noty(
```

Comment:
Shouldn't we check for response code like how we do it in javascript? For 401?

Reply:
I figured at this level, any error should be shown, but maybe it should be just 40x errors.

Reply:
I don't think we should show all errors, it might a db error or it could be any other generic exception when the form is submitted and this ticket is to deal with unauthorized users in dataset-submission